_includes/v21.1/cdc/url-encoding.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
+Parameters often contain special characters that need to be URI-encoded before they are included the changefeed's URI. Use Javascript's [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) function or Go language's [url.QueryEscape](https://golang.org/pkg/net/url/#QueryEscape) function to URI-encode the parameters. Other languages provide similar functions to URI-encode special characters.
```

Comment:
Should we tell users to always URI-encode their parameters?  The first sentence makes it seem more optional.